tests/python/gaia-ui-tests/gaiatest/gaia_test.py

```diff
@@ -860,6 +862,7 @@ class GaiaTestCase(MarionetteTestCase, B2GTestCaseMixin):
     def __init__(self, *args, **kwargs):
         self.restart = kwargs.pop('restart', False)
         self.locale = kwargs.pop('locale')
+        self.capture = kwargs.pop('capture', False)
```

Comment:
I guess this line overrides the default [already defined here](https://github.com/mozilla-b2g/gaia/pull/33815/files#diff-cb4505a1afd5a144fa750cd8b70aedf2R27), I'd remove it.